tests/test_pydantic_settings.sh

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env bash
+
+set -x
+set -e
+
+cd pydantic-settings
```

Comment:
This will work only when invoking from the root project dir.

`cd "$(dirname $0)/../pydantic-settings"` will allow to call this script from anywhere and always get the same result.

Also, using `pushd <dir>` at the start of the script and `popd` at the end will allow returning to the original director after script has finished running.

Reply:
You just do `pushd "$(dirname $0)/../pydantic-settings"` at the start and `popd` at the end, no need in any other `cd` invocations. This way it will always work no matter what the working dir is and it will return it to the original one.